site/en/blog/third-party-scripts/index.md

```diff
@@ -0,0 +1,264 @@
+
+3.  **Give ***third-parties*** incentives to reduce their performance impact via better transparency efforts.**
+
+    Third-party developers currently lack the visibility required to understand the impact of their scripts on sites at large. We plan to address this issue and equip these providers with tools to analyze their impact and compare it with other products in the market that provide the same value. We also want to help them use the data to diagnose what causes the impact so that they can mitigate it upstream. We will have to call out all third-parties, including those authored by Google, to succeed.
+
+## Goals
+
+We will know we have contributed significantly to reducing the impact of third-parties on CWV when we have achieved the following goals.
+
+-   50% decrease in partner origins who struggle to hit Core Web Vitals because of third-parties.
+-   50% of third-parties identified as most problematic for Core Web Vitals have a well-lit path solution.
+
+## Challenges
+
+An effort of this magnitude is not without challenges. Some of the key challenges we have to consider are.
+
+-   Third-parties are a cross-cutting problem involving ads, analytics, tag management, utilities, and many others. Each area requires consideration of a unique set of requirements and trade-offs. For example,
+-   The decision to optimize the loading of ads depends on a trade-off between revenue and user experience. Too early—they block valuable content; too late—the user would miss seeing them.
+-   Analytics scripts add to the page weight but provide valuable data about user actions to the business.
+
+We hope to partner with various categories of third-parties, grasp the nuances involved, resolve trade-offs, and develop incentives that work for all. We realize that we have to work separately with entities in every area for our strategy to be effective. This includes our internal partners such as Google Tag Manager, Google Ads, and YouTube.
+
+2.  We want to provide deeper attribution to both site developers and third-party developers. This requires a conscientious effort where we identify what data is most relevant in measuring impact, attribute it accurately and granularly, and provide the right path forward. Ultimately, the calculation of how a given third-party performs against its competition should be transparent to all.
+
+3.  We propose to enhance Chromium so that it can apply optimizations that the user has opted in for. A valuable change becomes available as a standard across all browsers, but it takes time. For example, the [loading](https://developer.mozilla.org/docs/Web/HTML/Element/img#attr-loading) attribute for images and iframes has been available in Chrome/Edge since 2019, but became available in Safari only in 2022. Until a feature is standardized, third-party users will have to ensure that they have optimized for other browsers too. We will help by highlighting this in our developer tools.
```

Comment:
```suggestion
3.  We propose to enhance Chromium so that it can apply optimizations that the user has opted in for. A valuable change becomes available as a standard across all browsers, but it takes time. For example, the [`loading`](https://developer.mozilla.org/docs/Web/HTML/Element/img#attr-loading) attribute for `<img>` and `<iframe>` elements has been available in Chrome/Edge since 2019, but became available in Safari only in 2022. Until a feature is standardized, third-party users will have to ensure that they have optimized for other browsers too. We will help by highlighting this in our developer tools.
```